test/e2e/addonmanagement_test.go

```diff
@@ -163,6 +163,22 @@ var _ = ginkgo.Describe("Enable addon management feature gate", ginkgo.Ordered,
 			return fmt.Errorf("the managedClusterAddon %s should be deleted", addOnName)
 		}).ShouldNot(gomega.HaveOccurred())
 
+		gomega.Eventually(func() error {
```

Comment:
addon was deleted, there should not be any work here, if we want to check if any work exists, I recommend not using Eventually.

Reply:
ok. done.